default.json5

```
@@ -144,6 +144,16 @@
       groupName: 'non-major netty dependencies',
       groupSlug: 'non-major-netty'
     },
+    {
+      description: 'Allow Netty updates for projects that have already migrated to version >= 4.2',
+      matchPackageNames: [
+        'io.netty:*'
+      ],
+      matchCurrentVersion: '[4.2.0.Final,)',
+      allowedVersions: '/^4\\.[2-9]+\\.[0-9]+(\\.[0-9A-Za-z]+)?$/',
```

I think simpler regex might be enough:
```suggestion
      allowedVersions: '/^4\\.[0-9]+\\.[0-9]+\\.Final$/',
```
* I've never seen another suffix than `.Final`.
* This would also cover `4.10.x.Final` and newer (not that this will come anytime soon).
* This also would allow `4.1.127.Final`, but Renovate will not downgrade to a previous minor version.